### ISSUE TITLE
Add Applications to custom output fields

### ISSUE THREAD

**Issue by kaihoffman** [civo/cli#164]
Currently the fields you can output by in custom output are:

```
	* ID
	* Code
	* Name
	* Nodes
	* Size
	* CPUCores
	* RAMMegabytes
	* DiskGigabytes
	* Status
	* KubernetesVersion
	* APIEndPoint
	* MasterIP
	* DNSEntry
```

I think it would be useful to add `Applications` or `InstalledApplications` to this set of options given that the cluster output does return the applications, and it can be parsed through the json option already.

**Comment by haardikdharma10** [comment]
@kaihoffman I _think_ we do output `InstalledApplications`. When I run `civo k3s show CLUSTER-NAME -o custom` it gives me the following - 

```
Pool (2b2460):
+---------------------------------------+---------------+--------+-----------------+-----------+----------+---------------+
| Name                                  | IP            | Status | Size            | Cpu Cores | RAM (MB) | SSD disk (GB) |
+---------------------------------------+---------------+--------+-----------------+-----------+----------+---------------+
| k3s-new-lb-7960-6aed69-node-pool-adc3 |               | ACTIVE | g4s.kube.medium |         2 |     4096 |            50 |
| k3s-new-lb-7960-6aed69-node-pool-d9ff | 74.220.18.163 | ACTIVE | g4s.kube.medium |         2 |     4096 |            50 |
| k3s-new-lb-7960-6aed69-node-pool-e4a8 |               | ACTIVE | g4s.kube.medium |         2 |     4096 |            50 |
+---------------------------------------+---------------+--------+-----------------+-----------+----------+---------------+

Labels:
kubernetes.civo.com/node-pool=2b24605c-b115-4ca7-ab71-8ee0532edf14
kubernetes.civo.com/node-size=g4s.kube.medium


Applications:
+-------------------------+-----------+-----------+--------------+
| Name                    | Version   | Installed | Category     |
+-------------------------+-----------+-----------+--------------+
| Traefik-v2-loadbalancer |       2.5 | true      | architecture |
| metrics-server          | (default) | false     | architecture |
+-------------------------+-----------+-----------+--------------+
```

I don't know if I misunderstood the issue here? 

`civo k3s show --help` indeed lists only 
```
	* ID
	* Code
	* Name
	* Nodes
	* Size
	* CPUCores
	* RAMMegabytes
	* DiskGigabytes
	* Status
	* KubernetesVersion
	* APIEndPoint
	* MasterIP
	* DNSEntry
```
Maybe we need to document `InstalledApplications` here?

**Comment by kaihoffman** [comment]
Yes, it would be good to have InstalledApplications as an option for an output field. So that if I ran `civo k8s show mycluster -o custom -f InstalledApplications` it would show me just the list of apps.

However, it looks like the custom output for existing fields outputs excess information too, on the current version 1.0.24:

```console
$ civo k3s show test -o custom -f APIEndPoint
https://74.220.21.244:6443

Pool (725f1b):
+-------------------------------------+---------------+--------+-----------------+-----------+----------+---------------+
| Name                                | IP            | Status | Size            | Cpu Cores | RAM (MB) | SSD disk (GB) |
+-------------------------------------+---------------+--------+-----------------+-----------+----------+---------------+
| k3s-test-a94a-8c03c9-node-pool-63d4 |               | ACTIVE | g4s.kube.medium |         2 |     4096 |            50 |
| k3s-test-a94a-8c03c9-node-pool-7ef5 | 74.220.21.244 | ACTIVE | g4s.kube.medium |         2 |     4096 |            50 |
| k3s-test-a94a-8c03c9-node-pool-c236 |               | ACTIVE | g4s.kube.medium |         2 |     4096 |            50 |
+-------------------------------------+---------------+--------+-----------------+-----------+----------+---------------+

Labels:
kubernetes.civo.com/node-pool=725f1bfb-3cea-4b4f-8dfc-28d299c62cdd
kubernetes.civo.com/node-size=g4s.kube.medium


Applications:
+---------------------+-----------+-----------+--------------+
| Name                | Version   | Installed | Category     |
+---------------------+-----------+-----------+--------------+
| metrics-server      | (default) | false     | architecture |
| Traefik-v2-nodeport |       2.5 | true      | architecture |
| Flux                | Latest    | true      | ci_cd        |
+---------------------+-----------+-----------+--------------+
```

You can see there's Pool, Labels and Applications I didn't ask for.

**Comment by kaihoffman** [comment]
Oh, and for `InstalledApplications` I guess I would expect the output to _just_ be the applications, rather than:

```console
)$ civo k3s show test -o custom -f InstalledApplications
InstalledApplications

Pool (725f1b):
+-------------------------------------+---------------+--------+-----------------+-----------+----------+---------------+
| Name                                | IP            | Status | Size            | Cpu Cores | RAM (MB) | SSD disk (GB) |
+-------------------------------------+---------------+--------+-----------------+-----------+----------+---------------+
| k3s-test-a94a-8c03c9-node-pool-63d4 |               | ACTIVE | g4s.kube.medium |         2 |     4096 |            50 |
| k3s-test-a94a-8c03c9-node-pool-7ef5 | 74.220.21.244 | ACTIVE | g4s.kube.medium |         2 |     4096 |            50 |
| k3s-test-a94a-8c03c9-node-pool-c236 |               | ACTIVE | g4s.kube.medium |         2 |     4096 |            50 |
+-------------------------------------+---------------+--------+-----------------+-----------+----------+---------------+

Labels:
kubernetes.civo.com/node-pool=725f1bfb-3cea-4b4f-8dfc-28d299c62cdd
kubernetes.civo.com/node-size=g4s.kube.medium


Applications:
+---------------------+-----------+-----------+--------------+
| Name                | Version   | Installed | Category     |
+---------------------+-----------+-----------+--------------+
| metrics-server      | (default) | false     | architecture |
| Traefik-v2-nodeport |       2.5 | true      | architecture |
| Flux                | Latest    | true      | ci_cd        |
+---------------------+-----------+-----------+--------------+
```

(which is not outputting the installed applications as the first output, just the field name(?) and then the excess information from the above comment)

**Comment by haardikdharma10** [comment]
Fixed in #209

**Comment by kaihoffman** [comment]
Confirmed that both of the above issues are resolved in version 1.0.25. Thanks @haardikdharma10 !
